modules/nested-tabs/assets/scss/frontend.scss

```diff
@@ -141,8 +146,7 @@
 					}
 
 					svg {
-						width: var( --n-tabs-icon-size );
-						height: var( --n-tabs-icon-size );
+						height: var( --n-tabs-svg-icon-size );
```

Comment:
Why has this been removed?

Reply:
If we set `initial` we receive a huge SVG image.
We have to supply a fixed size, to begin with.
At least that is what I saw has been done in other places that use SVG icons

Reply:
I have implemented the changes from your PR.